src/components/Restoration/CorrectCompanyName.vue

```diff
@@ -0,0 +1,85 @@
+<template>
```

Comment:
A straight copy from Edit UI. (Ditto the other CorrectXXX components.)

I intentionally didn't change the name, so we can maybe extract this into the shared components repo later.

Reply:
Makes sense!